doc/boot_process.adoc

```diff
@@ -54,7 +54,7 @@ keys are held, it will instead show the recovery menu.
 
 * Volume up
 * Volume down
-* Left shift
+* Left alt
```

Comment:
(Note to self: ensure once this PR is about to be merged that this is sync'd with the actual changes from `boot/lib/hal/recovery.rb`...)